cypress/e2e/Transcript/Access/BoardAndDoBIsRequiredToSomeBoards.feature

```diff
@@ -0,0 +1,16 @@
+Feature: The user is required to complete their profile to see the transcripts
+  Background:
+    Given I am a registered user
+    And I have selected that I have a medical board
+    And I did not provide the required <MedicalBoardInformation>
+
+  Scenario: user has empty board or date of birth details
```

Comment:
We should mention what happens with the transcript access

Reply:
which line you mean?

Reply:
Line 7, the scenario title.

`Scenario: user having empty board or date of birth details cannot access the transcript page`